CMakeLists.txt

```diff
@@ -49,6 +49,15 @@ endif ()
 
 set (CMAKE_CXX_STANDARD 11)
 
+# Use C++11
+if (CMAKE_VERSION VERSION_LESS "3.1")
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+        set (CMAKE_CXX_FLAGS "--std=gnu++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
Travis is still using gcc 4.6.3, I'm not sure how much of c++14 is supported in that?